src/installer/image/config.go

```diff
@@ -20,5 +20,5 @@ const (
 
 var (
 	CacheDir = filepath.Join(dtcsi.DataPath, "cache")
-	log      = logger.NewDTLogger().WithName("oneagent-image-installer")
+	log      = logger.Factory.GetLogger("oneagent-image-installer")
```

Comment:
to be consistent  with `log = logger.Factory.GetLogger("oneagent-url")`
```suggestion
	log      = logger.Factory.GetLogger("oneagent-image")
```

OR it can remain the same and update the `log = logger.Factory.GetLogger("oneagent-url")` to `log = logger.Factory.GetLogger("oneagent-url-installer")` 😉 

it feels weird that in the webhook we have reeally log logger names but this was considered too long.
For example `mutation-webhook-pod-oneagent` is a bit of a mouthful. (it could be `mutation-pod-oneagent` or just `mutation-oneagent` as those logs will only be seen in context of the webhook)

Reply:
Just shortened logger names for mutator.
Currently used logger names are:
```
$ ag --nofilename --nobreak "GetLogger" | grep -Eo "\".*\"" | sort | uniq
"activegate-customproperties"
"activegate-statefulset"
"activegate-statefulset-builder"
"automatic-api-monitoring"
"certificate-watcher"
"csi-appvolume"
"csi-driver"
"csi-gc"
"csi-hostvolume"
"csi-metadata"
"csi-provisioner"
"docker-config"
"dtclient"
"dynakube"
"dynakube-activegate-authtoken"
"dynakube-activegate-capability"
"dynakube-activegate-proxy"
"dynakube-api"
"dynakube-connectioninfo"
"dynakube-istio"
"dynakube-oneagent"
"dynakube-pullsecret"
"dynakube-status"
"dynakube-version"
"ingestendpoint"
"initgeneration"
"main"
"mutation"
"mutation-dataingest"
"mutation-namespace"
"mutation-oneagent"
"mutation-webhook"
"namespace-mapper"
"nodes"
"oneagent-image"
"oneagent-symlink"
"oneagent-url"
"oneagent-zip"
"processmoduleconfig"
"standalone-init"
"test-manager"
"test-secret"
"validation-webhook"
"version"
"webhook-certificates"
```